_saas-integrations/shopify/v1/shopify-v1.md

```diff
@@ -81,6 +81,16 @@ setup-steps:
       4. Click {{ app.buttons.finish-int-setup }}.
   - title: "track data"
 
+# -------------------------- #
+#     Replication Details    #
+# -------------------------- #
+
+replication-sections:  
+  - title: "Order Refunds table replication"
+  - anchor: "order-refunds"
+  - content: |
+      The Order Refunds table is a Key-based Incremental file that gets queried for every single order ID. If you have this table selected for replication, the process can potentially be very slow depending on how many refunds exist in the table. With Stich's replication process, this could cause other tables to not be replicated for days to weeks at a time. To ensure timely replications of your other selected tables, consider creating a separate integration for only the Order Refunds table.
```

Comment:
I think the first sentence is a bit jargony - how can you simplify this and make the language more relatable?

This may also have API quota implications, which is worth mentioning.